helpers/season_helper.py

```diff
@@ -74,3 +75,16 @@ def stop_build_datetime_est(year=datetime.now().year):
     def stop_build_datetime_utc(year=datetime.now().year):
         """ Converts stop_build_date to a UTC datetime """
         return SeasonHelper.stop_build_datetime_est(year).astimezone(UTC)
+
+    @staticmethod
+    def first_event_datetime_utc(year=datetime.now().year):
+        """ Computes day the first in-season event begins """
+        events = event_query.EventListQuery(year).fetch()
```

Comment:
I think we might already have an index for querying events by start date and filtering for only official events, if we wanna do that. But I don't care much either way 🤷‍♂